### ISSUE TITLE
Lighthouse accessibility score hurt by no label for sign out button

### ISSUE THREAD

**Issue by Connor-Skudlarek** [Connor-Skudlarek/nextjs-dashboard#32]
Add a screen reader accessibility name for the button.

**Comment by Connor-Skudlarek** [comment]
This and an issue with the links having no label have now been corrected.
